### ISSUE TITLE
SQL query results not displayed when executed from GIST

### ISSUE THREAD

**Issue by karim-slaoui** [microsoft/vscode-mssql#18544]
- MSSQL Extension Version: 1.27.0
- VSCode Version: 1.96.2
- OS Version: Windows Server 2022 - 21H2 - Build 20348.230

Steps to Reproduce:

1. Install MSSQL extension
2. Connect to DB
3. Create New File named XYZ.sql and save locally
4. Type and run query > result is displayed
![Image](https://github.com/user-attachments/assets/c106c2ae-2a8e-42ca-a9f3-8893f49c49f1)

5. Copy query into new GIST and run from gist file > result is not displayed
![Image](https://github.com/user-attachments/assets/d7c2d90c-4caf-4bf5-9163-8a5812adeb39)

6. Copy query into GitHub repository and run from there > result is displayed
![Image](https://github.com/user-attachments/assets/d97c6d40-c69d-4247-aa3b-7c8f66227faa)

In summary, result is displayed if executed from locally saved file or from a github repo, but not from gist (yes, the gist is public, not secret)

**Comment by aasimkhan30** [comment]
Fixed by #19784
